src/containers/UserToolkit/index.js

```diff
@@ -0,0 +1,287 @@
+      govtrack_id: currentTownHall.govtrack_id || null,
+      mocDataPath: peopleDataUrl,
+      thp_id: currentTownHall.thp_id || null,
+      memberId: currentTownHall.govtrack_id || currentTownHall.thp_id,
+      uid,
+      userDisplayName,
+    };
+    currentTownHall.meetingType = 'No Events';
+    submitMetaData(metaData);
+    this.success();
+    return this.resetAll();
+  }
+
+  showModal(type, link) {
+    if (type === 'add') {
+      this.setState({ addVisible: true });
+    } else {
+      this.props.setSelectedLink(link);
+      this.setState({ editVisible: true });
+    }
+  };
+
+  handleCancel(type) {
+    if (type === 'add') {
+      this.setState({ addVisible: false });
+    } else {
+      this.setState({ editVisible: false });
+    }
+  };
+
+  handleDeleteLink() {
+    let payload = {
+      moc_id: this.props.selectedMoc.govtrack_id,
+      link_id: this.props.selectedLink.id,
+    }
+    this.props.deleteMemberLink(payload);
+  }
+
+  handleCreate() {
+    const form = this.addFormRef.props.form;
+    form.validateFields((err, values) => {
+      if (err) {
+        return;
+      }
+      this.props.addMemberLink({
+        member_id: this.props.selectedMoc.govtrack_id,
+        link_title: values.title,
+        link_url: values.url
+      })
+      form.resetFields();
+      this.setState({ addVisible: false });
+    });
+  };
+
+  handleUpdate() {
+    const form = this.editFormRef.props.form;
+    form.validateFields((err, values) => {
+      if (err) {
+        return;
+      }
+      let payload = {
+        moc_id: this.props.selectedMoc.govtrack_id,
+        link_id: this.props.selectedLink.id,
+        linkInfo: {
+          link_title: values.title,
+          url: values.url
+        }
+      }
+      this.props.editMemberLink(payload);
+      form.resetFields();
+      this.updateSuccess();
+      this.setState({ editVisible: false });
+    });
+  }
+
+  saveAddFormRef(formRef) {
+    this.addFormRef = formRef;
+  };
+
+  saveEditFormRef(formRef) {
+    this.editFormRef = formRef;
+  };
+
+  renderMocBtns() {
+    if (this.props.userMocs) {
+      return (
+        <MemberButtons
+          selectMoc={this.selectMoc}
+          selectedMoc={this.props.selectedMoc}
+          userMocs={this.props.userMocs}
+        />
+      );
+    }
+  }
+
+  renderMemberLinks() {
+    if (this.props.userMocs) {
+      return (
+        <LinksList
+          selectedMoc={this.props.selectedMoc}
+          memberlinks={this.props.selectedMemberLinks}
+          showModal={this.showModal}
+          showConfirm={this.showConfirm}
+          handleSubmit={this.handleNoEventSubmit}
+          handleAutoFillMember={this.handleAutoFillMember}
+        />
+      );
+    }
+  }
+
+  render() {
+    return (
+      <div>
+        <Card className="user-member-card" title={<span id="submitted-meta-data"><span>You've submitted: </span><span id="submitted-total">0</span><span> event(s)</span></span>}>
```

Comment:
Look at my most recent changes. This should now be coming from state.

Reply:
Ok, i've update eventCount received from state, thx!